documentation/src/components/DocsIframe/index.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+/**
+ * Displays generated HTML docs in an iframe.
+ *
+ * @param {{
+ *   docFolder: string,
+ *   width?: string,
+ *   height?: string
+ * }} props
+ */
+function DocsIframe({ docFolder, width = "100%", height = "600px" }) {
```

Comment:
This is awesome! I might suggest changing the name to be a little bit more intuitive. Maybe 
```suggestion
function InlineDocs({ docFolder, width = "100%", height = "600px" }) {
```
or
```suggestion
function EmbedDocs({ docFolder, width = "100%", height = "600px" }) {
```
Some students may be unfamiliar with `iframe` as a term, and I'd like to keep the docs template's usage to not require any web knowledge.